big_tests/test.config

```diff
@@ -241,6 +241,7 @@
   workers = 10"}]},
     {pgsql_cets,
      [{dbs, [redis, pgsql]},
+      {mnesia_or_cets_backend, cets},
```

Comment:
I agree that we should have an option like this - we could even call it `volatile_database` like the helm chart does.

However, I think that when adding this, we should remove all the `..._backend` options below. Otherwise sometimes we would use this option, and sometimes the per-module options, which becomes really hard to follow.

Reply:
Update: after merging https://github.com/esl/MongooseIM/pull/4155 we should use `ct_helper:get_internal_database/0` instead of instroducing this option.